handsontable/src/pluginHooks.js

```diff
@@ -1454,18 +1460,17 @@ const REGISTERED_HOOKS = [
    *
    * ::: only-for react
    * ```jsx
-   * // To disregard a single row, remove it from the array using data.splice(i, 1).
+   * // To disregard a single row or column, remove it from copied dataset using `remove` method on the object from the first callback argument.
```

Comment:
It seems the documentation is out of date. There is no `remove` method.

Reply:
Thanks, every wrong docs should be changed.